packages/esm-patient-chart-app/src/ui-components/patient-chart.component.tsx

```diff
@@ -19,15 +20,17 @@ interface PatientChartParams {
 const PatientChart: React.FC<RouteComponentProps<PatientChartParams>> = ({ match }) => {
   const { patientUuid, view, subview } = match.params;
   const [loading, patient] = useCurrentPatient(patientUuid);
+  const sessionUser = useSessionUser();
   const state = useMemo(() => ({ patient, patientUuid }), [patient, patientUuid]);
   const { active } = useWorkspace();
 
   const mainClassName = `
```

Comment:
Should that have new lines in there? I'd say let's make it a single line: ``const mainClassName = `omrs-main-content ${styles.chartContainer}`;``.